### PR TITLE
feat: Group call view - Part 2 :  grid indicator (AR-1148)

### PR DESCRIPTION
**Pull request by ohassine**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1148" title="AR-1148" target="_blank">StoryAR-1148</a>  View all call participants during the call (grid)
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Description

A follow up PR of #685

In this PR, we are updating the design of grid indicator.(see screenshot bellow)

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

Start a group with more than 8 participants to see the pager indicator.

### Attachments (Optional)

Wire 2022-06-23 at 12_19 PM

 
----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-700/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3701315)(3701315)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3701315)(3701315)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/ui/calling/common/VerticalCallingPager.kt [hunk 1]
```diff
@@ -59,12 +64,26 @@ fun VerticalCallingPager(
...
+                    VerticalPagerIndicator(
+                        modifier = Modifier.padding(4.dp),
+                        pagerState = pagerState,
+                        activeColor = colorsScheme().callingActiveIndicator,
+                        inactiveColor = colorsScheme().callingInActiveIndicator,
+                        inactiveBorderColor = colorsScheme().callingInActiveBorderIndicator,
+                        inactiveBorderWidth = 2.dp,
+                        indicatorHeight = 12.dp,
+                        indicatorWidth = 12.dp,
+                        spacing = 6.dp,
```
Comment (alexandreferris) [inline app/src/main/kotlin/com/wire/android/ui/calling/common/VerticalCallingPager.kt:83]: maybe we can use `dimensions().spacing12x` and `dimensions().spacing6x` here

**Review by alexandreferris — APPROVED** [review]
Looking great! I just have a small suggestion that is to remove hardcoded values and add its correct dimensions.

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-700/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3709326)(3709326)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3709326)(3709326)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-700/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3709476)(3709476)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3709476)(3709476)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-700/5/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3709580)(3709580)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3709580)(3709580)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [6](https://10.10.124.134/job/wire-android-reloaded/job/PR-700/6/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3709698)(3709698)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3709698)(3709698)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [7](https://10.10.124.134/job/wire-android-reloaded/job/PR-700/7/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3711143)(3711143)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3711143)(3711143)-dev-release.apk)

**Pull request merged**